src/inject/generic.js

```diff
@@ -5,7 +5,7 @@ import SettingsController from '../main/utils/Settings';
 global.Settings = new SettingsController();
 Settings.uncouple();
 
-require(`./${process.platform}`);
+require(`./${process.platform === 'win32' ? 'win32' : 'darwin'}`);
```

Comment:
I'm not sure this is what we want?

This import darwin (osx) based injections when building for `linux`.  Although this may work now it is counter intuitive, I think we should leave as is and have a `"linux"` folder that might possibly import things from the `darwin` folder to avoid code duplication.  Then it makes more sense